src/locale/misc/smartSearch/en.yml

```diff
@@ -79,6 +80,27 @@ random:
     examples:
         one: Randomly add 20 people in the organization.
         two: Randomly remove 15% of the people in the organization.
+survey_response:
+    inputString: "{addRemoveSelect} people whose response to {surveySelect} ({questionSelect}) {matchSelect} {freeTextInput}"
```

Comment:
I think `{freeTextInput}` should be enclosed in quotation marks which will make it easier to read in the sentence.